users/peterfalken/peterfalken.c

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 Peter.Falken (@PeterFalken)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "peterfalken.h"
+
+// Process custom keycodes
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case KC_LAUNCHPAD:
+            if (record->event.pressed) {
+                host_consumer_send(0x2A0);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;  // Skip all further processing of this key
+        default:
+            return true;  // Process all other keycodes normally
+    }
+}
```

Comment:
This is in core, and isn't needed anymore.

Reply:
I also just tried adjusting the `[_ADJUST]` layer on the `default` keymap for the `preonic/rev3_drop`, to check if the problem was on my userspace, but the results are the same, a key press for the keys defined as `KC_MCTL` or  `KC_LPAD` doesn't not send the consumer code either on the `default` keymap.
```
[_ADJUST] = LAYOUT_preonic_grid(
  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
  _______, QK_BOOT, DB_TOGG, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL,
  _______, _______, MU_NEXT, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______,
  _______, AU_PREV, AU_NEXT, MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MCTL, KC_LPAD
)
```

Reply:
Just tested, and can confirm they work (on a blackpill). 

There are a few potential reasons that it's not working properly. Namely, if extrakeys isn't enabled, if the repo is out of date, weird build issues.  `qmk clean -a` may help. 

But as this is in core, and I can verify that it does work locally...

Reply:
Removed `process_record_user` from userspace, will debug this on a separate commit.